packages/dialect-react-ui/components/Chat/navigation.ts

```diff
@@ -0,0 +1,37 @@
+import { MainRouteName, RouteName, ThreadRouteName } from './constants';
+import type { RouterContextValue } from '../common/providers/Router';
+
+type NavigateFn = RouterContextValue['navigate'] | undefined;
```

Comment:
just wonder, why undefined?

Reply:
Not all windows have navigations at the moment, which means not all have `navigate`.